CHANGELOG.rst

```diff
@@ -8,6 +8,44 @@
 
 .. towncrier release notes start
 
+Pytest 3.4.2 (2018-03-04)
+=========================
+
+Bug Fixes
+---------
+
+- Removed progress information when capture option is ``no``. (`#3203
+  <https://github.com/pytest-dev/pytest/issues/3203>`_)
+
+- Refactor check of bindir from ``exists`` to ``isdir`` regarding `#3241
+  <https://github.com/pytest-dev/pytest/issues/3241>`_. (`#3241
```

Comment:
this has gotten a accidental link duplicate, due to the changelog source entry, we can fix here or in the soruce

Reply:
Good catch, thanks! Fixed it here directly.